roles/ngi_pipeline/tasks/main.yml

```diff
@@ -1,22 +1,22 @@
 ---
 
-- name: fetch ngi_pipline from github 
+- name: Fetch ngi_pipeline from github 
   git: repo={{ ngi_pipeline_repo }} 
        dest={{ ngi_pipeline_dest }}
        version={{ ngi_pipeline_version }}
        force=yes
   tags: ngi_pipeline 
 
 
-- name: check if ngi virtual env exists 
-  shell: conda env list | awk '{print $1}' | egrep -e "^NGI$"
-  register: ngi_venv_exists
+- name: "Check if {{ NGI_venv_name }} virtual env exists" 
+  shell: "conda env list | awk '{print $1}' | egrep -e ^{{ NGI_venv_name }}$"
+  register: "{{ NGI_venv_name }}_venv_exists"
```

Comment:
Does this work? I.e have you verified that the `{{ NGI_venv_name }}` will be expanded in `register`? Because it didn't use to work in `name`, at least for older versions of Ansible.